### PR TITLE
Fixed File.exists? issue

### PR DESCRIPTION
**Pull request by saviokmua**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,4 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+.idea
```
Comment (ojab-test) [inline .gitignore:18]: Newline is missing

Comment (ojab) [inline .gitignore:18]: Newline is missing

**Comment by ojab** [comment]
ruby-3.2 removed `File#exists?` & `Dir#exists?` that were deprecated since ruby-2.1, so on ruby-3.2 rspec fails with 
```
NoMethodError:
  undefined method `exists?' for File:Class
# /usr/local/bundle/gems/watir-rspec-3.0.0/lib/watir/rspec/html_formatter.rb:19:in `initialize'
```

File: .gitignore (hunk 1 above)
Comment (jarmo) [inline .gitignore:18]: Editor-specific files/directories should not be part of this repo `.gitignore` file. Use global `.gitignore` on your machine instead if you're using IDEA and don't want to see its files as changed files.

**Comment by aditya-qapitol** [comment]
which version of ruby should be used to until this is fixed ?

**Comment by jarmo** [comment]
Released `watir-rspec` with version 4.0.0.

**Pull request closed without merging**